app/[blog]/new/page.js

```diff
@@ -79,7 +77,7 @@ const NewBlogPost = () => {
 
   return (
     <div className={styles.container}>
-      <form onSubmit={handleSubmit}>
+      <div>
```

Comment:
is there a way to keep the element as form? I used form because I wanted the built-in validation. Save time on checking and error indications.

Reply:
ok, changed it back and added some error checking for the MD editor component, since it was allowing empty posts/comments before